scripts/restore.sh

```diff
@@ -200,6 +209,7 @@ restore_pane() {
 		else
 			new_session "$session_name" "$window_number" "$dir" "$pane_index"
 		fi
+		set_pane_title "$session_name" "$window_number" "$pane_index" "$pane_title"
```

Comment:
`set_pane_title` function effectively contains only a single line/single command, and it's called from just one place. 

My first thought was why not just inline it? If you think the function name makes it obvious what the cryptic tmux command does, we can also add a comment.

But, if you want to keep the function, that's fine too.

Reply:
It is a remnant of previous commits where I tried to call it when creating panes. I agree to just inline the command.